src/helpdesk_app/views.py

```diff
@@ -83,6 +83,9 @@ def autocomplete_search(request):
     if 'term' in request.GET:
         query = request.GET.get('term')
         category = request.GET.get('c', '')
+        # if no category was actually selected
+        if category == "Category search" or category == "Categoría":
```

Comment:
What if they change the Spanish translation?

Reply:
Hmm, is there a way to get the spanish translation that they have inputted?

Reply:
I believe when no category is selected, the value that’s sent is `””`. You can check this by looking at the source for `search.html`. So I believe that this conditional check will never actually run, can you double check this? If so, it can be removed entirely I think.

Reply:
The way that the category is retrieved is like this:
`var categories_elements = document.getElementsByClassName("search_category");
                    var selected_category = categories_elements[0].options[categories_elements[0].selectedIndex].text;`
Every time the user types something into the search bar, the current category is retreived from the list of categories in the dropdown. The code gets the text from the selected category so "Category search" will be an option.

I tried a different way of updating the category when it is changed by the user, but that didn't really work out.

Reply:
In the Javascript can you not refer to the select dropdown’s `value` instead of `text`?

The `text` would be “Category search” but the value would be `””` … I think this is better as it’ll always be independent of whatever the translation string is?

Reply:
oh wait yeah that works lol, thanks!